### PR TITLE
a lot of fixes

### PR DESCRIPTION
**Pull request by ivan-kiselev**
Make it able to become cluster instantly and work with fresh 2.2 ansible.

### CODE REVIEW

File: meta/.galaxy_install_info [hunk 1]
```diff
@@ -0,0 +1 @@
+{install_date: 'Sun Dec 11 09:05:32 2016', version: v0.0.2}
```
Comment (veger) [inline meta/.galaxy_install_info]: This file should not be part of the role (it is added by `ansible-galaxy`)

File: tasks/config-cluster.yml [hunk 2]
```diff
@@ -1,21 +1,36 @@
 ---
 # Configure RabbitMQ for cluster
 
+- name: ensure hostnames are in the host file
+  lineinfile:
+    dest: /etc/hosts
+    line: "{{ hostvars[item]['ansible_default_ipv4']['address'] }} {{ hostvars[item]['ansible_hostname'] }}"
+  with_items: "{{ groups['rabbit'] }}"
```
Comment (veger) [inline tasks/config-cluster.yml]: What if I do not have a hosts group called 'rabbit'? (For example when I want to install two RabbitMQ clusters)

File: tasks/config-cluster.yml [hunk 3]
```diff
@@ -1,21 +1,36 @@
 ---
 # Configure RabbitMQ for cluster
 
+- name: ensure hostnames are in the host file
+  lineinfile:
+    dest: /etc/hosts
+    line: "{{ hostvars[item]['ansible_default_ipv4']['address'] }} {{ hostvars[item]['ansible_hostname'] }}"
```
Comment (veger) [inline tasks/config-cluster.yml]: When dynamic IPs are used, for example in Amazon AWS, then this breaks your /etc/hosts file completely as the (new) IPs are added again and again.

File: tasks/config.yml [hunk 4]
```diff
@@ -1,17 +1,38 @@
 ---
 # Configure RabbitMQ
 
-- name: rabbitmq config file
-  template: src=etc/rabbitmq/rabbitmq.config.j2 dest=/etc/rabbitmq/rabbitmq.config owner=root group=root mode=0644
```
Comment (veger) [inline tasks/config.yml]: What happened to the config file? Forgot the re-add again? It is required for RabbitMQ instlalations/clusters...

**Review by veger — COMMENTED** [review]
I have some comments/remarks about this change

**Comment by homelessnessbo** [comment]
@veger yep, u r totaly right. Sorry for this, I will try to workaround dat issues in the nearest future.

File: tasks/config.yml (hunk 4 above)
Comment (alekseyl1992) [reply tasks/config.yml:5]: this actually is a severe issue, and that PR was already merged for some reason

config file is not being copied at all

**Pull request merged**